index.html

```diff
@@ -24,52 +24,49 @@ <h3 class="text-center" id="list-title">Lodges Near You:</h3>
         <div class="col-md-4" id="filters-list">
           <div class="form-check">
             <label class="form-check-label">
-              <input class="form-check-input" type="checkbox" data-id="1" value="Blue Lodge">
+              <input class="form-check-input" type="checkbox" onclick="handleClick()" data-id="1" value="Blue Lodge">
```

Comment:
Ok, good job. But we don't want to use inline JavaScript like this. Just add an event listener from your `main.js`.